src/node.cc

```diff
@@ -1383,27 +1384,28 @@ void AppendExceptionLine(Environment* env,
                      linenum,
                      sourceline_string);
   CHECK_GE(off, 0);
```

Comment:
Yes the custom snprintf should be replaced with something like this:
The _TRUNCATE negates the non-standard -1 behaviour and VS 2015 has
snprintf.

#if _MSC_VER < 1900int snprintf(char *buffer, size_t n, const char *format,
...){ va_list argp; int ret; va_start(argp, format); ret = _vscprintf(format,
argp); vsnprintf_s(buffer, n, _TRUNCATE, format, argp); va_end(argp); return
ret;}#endif


-- 

Kind regards,

Karl Skomski